package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "phaser3-rex-plugins",
-  "version": "1.1.71",
+  "version": "1.1.72",
```

Comment:
Huh, that's weird. I haven't changed this file. I guess npm changed it when I did `npm install`. 

Anyway, the change looks correct to me, so I guess it's worth to leave it 🤔